src/main/kotlin/com/github/patou/gitmoji/GitMojiConfig.kt

```diff
@@ -67,23 +74,28 @@ class GitMojiConfig(private val project: Project) : SearchableConfigurable {
             1 -> " "
             else -> textAfterUnicodeOptions[textAfterUnicode.selectedIndex]
         }
+        languagesConfig = languageOptions[languages.selectedIndex]
 
         val projectInstance = PropertiesComponent.getInstance(project)
+        val instance = PropertiesComponent.getInstance()
         projectInstance.setValue(CONFIG_DISPLAY_ICON, displayEmojiConfig)
         projectInstance.setValue(CONFIG_INSERT_IN_CURSOR_POSITION, insertInCursorPositionConfig)
         projectInstance.setValue(CONFIG_USE_UNICODE, useUnicodeConfig)
         projectInstance.setValue(CONFIG_INCLUDE_GITMOJI_DESCRIPTION, includeGitMojiDescriptionConfig)
         projectInstance.setValue(CONFIG_AFTER_UNICODE, textAfterUnicodeConfig)
+        instance.setValue(CONFIG_LANGUAGE, languagesConfig)
     }
 
     override fun reset() {
         val propertiesComponent = PropertiesComponent.getInstance(project)
+        val instance = PropertiesComponent.getInstance()
 
         displayEmojiConfig = propertiesComponent.getValue(CONFIG_DISPLAY_ICON, Gitmojis.defaultDisplayType())
         useUnicodeConfig = propertiesComponent.getBoolean(CONFIG_USE_UNICODE, false)
         insertInCursorPositionConfig = propertiesComponent.getBoolean(CONFIG_INSERT_IN_CURSOR_POSITION, false)
         includeGitMojiDescriptionConfig = propertiesComponent.getBoolean(CONFIG_INCLUDE_GITMOJI_DESCRIPTION, false)
         textAfterUnicodeConfig = propertiesComponent.getValue(CONFIG_AFTER_UNICODE, " ")
+        languagesConfig = instance.getValue(CONFIG_LANGUAGE, "en_US")
```

Comment:
Maybe reset to auto ? It's the default value.
```suggestion
        languagesConfig = instance.getValue(CONFIG_LANGUAGE, "auto")
```